dev/docker-compose.yaml

```diff
@@ -20,6 +20,8 @@ services:
       - jaeger
 
   jaeger:
-    image: jaegertracing/all-in-one:1.46.0
+    image: jaegertracing/all-in-one:1.50.0
+    environment:
+      - COLLECTOR_OTLP_ENABLED=true
```

Comment:
Is this variable really enabled? As I remember, it is now default value.

Reply:
It is not. I have forgotten to remove it.